### DISCUSSION TITLE
ubuntu use cookie is not ok

### DISCUSSION THREAD

**Discussion by boluoha** [projectdiscovery/nuclei#2475] [Q&A] [answered]
I use nuclei  to scan a login website,it's need add cookie!
I hava use it,for example :
```sh
nuclei -u http://www.abc.com/vulnerabilities/sqli/?id=1&Submit=Submit# -H "Cookie:security_level=0;PHPSESSID=k6e386...abflltq2;security=low"  -t /root/nuclei-templates/vulnerabilities/generic/error-based-sql-injection.yaml  ---json -irr -o 2022.json
```

I want to use, how to use it on Ubuntu 18.04 server!
thanks!

**Comment by forgedhallpass** [comment, marked as answer]
Hello @boluoha 

You need to put the URL between quotes because the `#` acts as comment in the terminal. You also need to change the `---json` to `-json`.

p.s. Don't share your session IDs publicly :)
